### PR TITLE
WT-6312 history store verification

### PR DESCRIPTION
**Pull request by keithbostic**
The only thing that's ugly here is setting `WT_CURSTD_IGNORE_TOMBSTONE` in the cursor so we don't skip over all the history-store records, otherwise this doesn't seem that unreasonable to me.

Of course, just discard this branch without further discussion if this isn't the way you want to go.

### CODE REVIEW

File: src/history/hs.c [hunk 1]
```diff
@@ -1414,7 +1414,7 @@ __wt_history_store_verify_one(WT_SESSION_IMPL *session)
      * in the history store.
      */
     memset(&hs_key, 0, sizeof(hs_key));
-    cursor->set_key(cursor, btree_id, &hs_key, 0, 0, 0, 0);
```
Comment (agorrod) [inline src/history/hs.c:1417]: Is this a straight-up bug in the current code?

Comment (keithbostic) [reply src/history/hs.c:1417]: Yes -- no bad effects though, just a typo.

File: test/format/hs.c [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        testutil_check(ret);
+
+        /*
+         * The history file has mostly tombstones, ignore them and retrieve the underlying values.
+         * We don't care about tombstones, but we do want to hit every key rather than skip over
+         * them. This is a rollback-to-stable flag we're using for our own purposes.
+         */
+        F_SET(cursor, WT_CURSTD_IGNORE_TOMBSTONE);
+
+        /* Search to the last-known location. */
```
Comment (tetsuo-cpp) [inline test/format/hs.c:81]: What happens if we hit the end of the history store? Isn't this going to keep searching for the last key, doing a next to get `WT_NOTFOUND`, rinse repeat?

Comment (keithbostic) [reply test/format/hs.c:81]: Yes, I think you're correct -- it's not guaranteed, but you're right most of the time, at the least. I've pushed a fix, would you please re-review, @tetsuo-cpp?

**Comment by keithbostic** [comment]
evergreen: retry

**Pull request merged**